Source/Scene/Scene.js

```diff
@@ -1444,7 +1493,7 @@ define([
 
         if (environmentState.isSunVisible) {
             environmentState.sunDrawCommand.execute(context, passState);
-            if (scene.sunBloom) {
+            if (scene.sunBloom && !scene._useWebVR) {
```

Comment:
Do you think this is a bug in Cesium?  In the browser?  At the least, we should add a comment about this.  I would rather fix it now if it is easy enough.